src/standard.js

```diff
@@ -1,11 +1,35 @@
-export default function standard(cost) {
+// emoji, attack, health, name, text, onSummon
+export function standard(cost) {
   const emoji = emojis[cost] || emojis[emojis.length - 1];
   return {
     cost: cost,
     emoji: emoji,
     attack: cost,
     health: cost,
-    name: `Standard ${cost}-${cost}`
+    name: `Standard ${cost}-${cost}`,
+    text: ""
+  };
+}
+//need action to change health or attack of a creature already made by game
+export function battlecry(cost) {
+  const emoji = emojis[cost] || emojis[emojis.length - 1];
+  return {
+    cost: cost,
+    // emoji: emoji,
+    attack: cost,
+    health: cost,
+    name: "Sunkeeper Tarim",
+    text: `Battlecry: Set all other minions' Attack and Health to 3`,
```

Comment:
same deal, I think I want to go with "On summon: X" rather than "Battlecry: X"

Reply:
and "creatures" rather than "minions"

i might give up on this before too long but let's stick with it for now